include/SFML/Graphics/RenderTexture.h

```diff
@@ -85,6 +85,16 @@ CSFML_GRAPHICS_API void sfRenderTexture_destroy(sfRenderTexture* renderTexture);
 ////////////////////////////////////////////////////////////
 CSFML_GRAPHICS_API sfVector2u sfRenderTexture_getSize(const sfRenderTexture* renderTexture);
 
+////////////////////////////////////////////////////////////
+/// \brief Tell if the render target will use sRGB encoding when drawing on it
+///
+/// \param renderTexture Render texture object
+///
+/// \return sfTrue if the render target use sRGB encoding, false otherwise
```

Comment:
To be more specific:
```suggestion
/// \return sfTrue if the render texture use sRGB encoding, false otherwise
```